app/cdap/components/RecipeManagement/ApplyRecipeView.tsx

```diff
@@ -217,7 +217,6 @@ export const ApplyRecipeView = ({
         showDivider={false}
         open={isOpen}
         dataTestId={'apply-recipe-drawer-widget'}
-        showHeaderSeparator={false}
```

Comment:
why change this file, is there any specific reason?

Reply:
In previous design we have separator in header, but as we don't have header separator  in v3 latest design we removed it